packages/libxcrypt.rb

```diff
@@ -1,36 +1,25 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Libxcrypt < Package
+class Libxcrypt < Autotools
   description 'Modern library for one-way hashing of passwords'
   homepage 'https://github.com/besser82/libxcrypt/'
-  version '4.4.18'
+  version '4.4.38'
   license 'LGPL-2.1+, public-domain, BSD and BSD-2'
   compatibility 'all'
-  source_url "https://github.com/besser82/libxcrypt/archive/v#{version}.tar.gz"
-  source_sha256 '3801f0263a8596b15ec466343fc1fdc4ad4ec7416c51e038a3528fd47f3be01a'
-  binary_compression 'tar.xz'
+  source_url "https://github.com/besser82/libxcrypt/releases/download/v#{version}/libxcrypt-#{version}.tar.xz"
+  source_sha256 '80304b9c306ea799327f01d9a7549bdb28317789182631f1b54f4511b4206dd6'
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '5be9496841308529004f767f689f9d0ad681700e12db213f2f94742aec99e1d3',
-     armv7l: '5be9496841308529004f767f689f9d0ad681700e12db213f2f94742aec99e1d3',
-       i686: 'c06e88bb39f787c294dbbdb5497887a6b58b24d586965ece1d9915dd3cd82e0c',
-     x86_64: '36f1ed8e9d2889c62c2eb4f9d98808fcae9400d376e1b6cd8367550998a8188c'
+    aarch64: 'da8f6564a1aed32e2d31c2a349cecd2754c6914325379a3f797c86b02765ac1f',
+     armv7l: 'da8f6564a1aed32e2d31c2a349cecd2754c6914325379a3f797c86b02765ac1f',
+       i686: '1335e04f868ecb914a00c9fed2746009d19f65b1a2961f47aebf2cc5634e346e',
+     x86_64: '5c6d9f5fdb8af6d2b86652e9cb51156fcc6a7f44d7db4f6f46ee826f3deb58b7'
   })
 
-  def self.build
-    system 'autoreconf -fi'
-    system "env CFLAGS='-flto=auto' \
-      CXXFLAGS='-flto=auto' LDFLAGS='-flto=auto' \
-      ./configure \
-      #{CREW_CONFIGURE_OPTIONS} \
-      --disable-static \
-      --enable-hashes=strong,glibc \
-      --enable-obsolete-api=no \
-      --disable-failure-tokens"
-    system 'make'
-  end
+  depends_on 'glibc' # R
 
-  def self.install
-    system "make DESTDIR=#{CREW_DEST_DIR} install"
-  end
+  conflicts_ok
```

Comment:
What package does this conflict with?

Reply:
glibc_build### where ### is the glibc version.

Reply:
The current libcrypt.so is corrupt.  Hence this update.